administration/src/util/serializeToCanonicalJson.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Makes typescript aware of the presence of a property in an object and let it narrow down the type, so that we can
+ * access the property.
+ */
+function hasProp<K extends PropertyKey>(data: object, prop: K): data is Record<K, unknown> {
+  return prop in data
+}
+
+/**
+ * Returns a serialization of the passed object according to RFC 8785 JSON Canonicalization Scheme (JCS).
+ *
+ * Taken & adjusted from https://github.com/erdtman/canonicalize/blob/master/lib/canonicalize.js
+ * Under Apache 2.0 License
+ *
+ * Modifications:
+ * 1) Throw, if undefined is passed to the function (instead of returning undefined).
+ * 2) Throw, if an item of an array is undefined or if it is a Symbol (instead of serializing null).
+ * 3) Throw, if the value of an object is undefined or if it is a Symbol (instead of skipping the entry).
+ */
+function serializeToCanonicalJson(object: unknown): string {
```

Comment:
I think we should port over the simple tests here: https://github.com/erdtman/canonicalize/blob/master/test/simpletests.js

Reply:
Done.